### PR TITLE
feat: pass strikethrough markdown for hint message on field

### LINKED ISSUE #5717: Support strikethrough markdown in widget hint text
**Is your feature request related to a problem? Please describe.**
There is markdown support in field hints for inline features like **bold**, _italic_, and links, but not strikethroughs. Niche use case but should be easy to do and fits in with the other features.

Example:
I'm using the hint to give the editors instructions on how to format an external link, showing the whole context and which parts to remove:
`https://www.linkedin.com/pulse/complex-world-early-interventions~~?trk=public_profile_article_view~~`

**Describe the solution you'd like**
Support adding Markdown strikethroughs `~~delete me~~`

**Additional context**
#5559 closed with #5584
@erezrokah gave this code as reference:
https://github.com/netlify/netlify-cms/blob/610421ea6722276e37fe3c6123386d20fbbac1cf/packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js#L333

### PR DESCRIPTION
**Pull request by JonasMatos0**
**Summary**

This adds the option to include strikethrough HTML in the hint message on a field. Closes #5717
The issue creator requested support for strikethrough Markdown

**Test plan**

Please guide me to the testing details if any for this modification. Thanks in advance.

**Checklist**

Please add a `x` inside each checkbox:

- [x] I have read the [contribution guidelines](../CONTRIBUTING.md).
- [x] Code is formatted via running `yarn format`.
- [x] Tests are passing via running `yarn test`.
- [x] The status checks are successful (continuous integration). Those can be seen below.

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js [hunk 1]
```diff
@@ -330,7 +331,8 @@ class EditorControl extends React.Component {
             {fieldHint && (
               <ControlHint active={isSelected || this.state.styleActive} error={hasErrors}>
                 <ReactMarkdown
-                  allowedElements={['a', 'strong', 'em']}
+                  plugins={[gfm]}
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js:334]: Per the [docs](https://github.com/remarkjs/react-markdown/tree/ce6c1a71c17280e753e54e919511cd8bafadf86e#use) this should be `remarkPlugins={[gfm]}`

File: packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js [hunk 2]
```diff
@@ -330,7 +331,8 @@ class EditorControl extends React.Component {
...
+                  allowedElements={['a', 'strong', 'em', 's']}
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js:335]: Per the [docs](https://github.com/remarkjs/react-markdown/tree/ce6c1a71c17280e753e54e919511cd8bafadf86e#use-a-plugin) this should be `allowedElements={['a', 'strong', 'em', 'del']}`

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @JonasMatos0, I added 2 comments.

Please note that even with my suggestion I couldn't get neither `~strikethrough~` nor `~~strikethrough~~` to work.

File: packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js (hunk 2 above)
Comment (JonasMatos0) [reply packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js]: My bad. Updated the reference.

**Comment by erezrokah** [comment]
Hi @JonasMatos0, can you share a sample `config.yml` where you got the strikethrough to work?

**Review by erezrokah — APPROVED** [review]
Thanks for digging into this @JonasMatos0, sorry for the late approval.

**Pull request merged**